test/test_electro.jl

```diff
@@ -0,0 +1,40 @@
+@testset "electrostatics.jl" begin
```

Comment:
The string is commonly just a description, doesn't need to match the filename. I'm don't particularly care though...